agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/jdbc/DBInfo.java

```diff
@@ -30,4 +30,6 @@
   private final String db;
   private final String host;
   private final Integer port;
+
+  public static class Builder {}
```

Comment:
Without this explicit declaration javadoc generation for `JDBCConnectionUrlParser` fails with 
```
error: cannot find symbol
  abstract DBInfo.Builder doParse(String jdbcUrl, final DBInfo.Builder builder);
```